### ISSUE TITLE
Build command to respect the main file set in elm-package.json

### ISSUE THREAD

**Issue by nmehta6** [deadfoxygrandpa/Elm.tmLanguage#96]
Currently, building (not running) only executes elm-make command on the current file. It does not respect  if you have set the main field in elm-package.json. It would be great if the build command respected the main field. This way, you can build the whole project (which I find more desirable than building the current file). If the main field is not set, it should just build the current open file. Please let me know if you would be willing to accept a PR. Thanks.

**Comment by deadfoxygrandpa** [comment]
I agree with this idea. I can make it myself, unless you want to submit the PR. There's an ElmProject class which can locate the Main module and build that instead, it should be a pretty easy fix.

**Comment by nmehta6** [comment]
Sorry experimented with the python code for bit, but didn't get far since I am not too familiar with the language. If you can make this change, that would be great!
